config/environments/development.rb

```diff
@@ -50,4 +50,7 @@
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
   config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+
+  # FreshService URL
+  config.fresh_sevice_new_ticket_url = "https://sanger.freshservice.com/a/tickets/new"
```

Comment:
Just to give you a reminder - I am sure you have updated the deployment project with fresh_service env variables as well. :)

Reply:
Yes :) here is the PR https://github.com/sanger/deployment/pull/490